Makefile

```diff
@@ -1,3 +1,6 @@
 build-wrapper:
 	cd wrapper && cargo build --release --target wasm32-wasip1
 	cp wrapper/target/wasm32-wasip1/release/wrapper.wasm descriptors/
+
+unit:
```

Comment:
I prefer `unit-test` or `run-unit-tests`, or to drop this completely as `go test ./...` is easy enough on its own.